docs/pages/Builder.md

```diff
@@ -96,5 +96,12 @@ Since string replacement is a common build step, it can be useful in different c
 
 Available processors are listed [in the API reference](https://sap.github.io/ui5-tooling/api/module-@ui5_builder.processors.html).
 
+## Source Map support
+
+Source Maps will be supported by the upcoming v3.0.0 version which is currently in development.
+Files will be generated automatically for all bundles.
```

Comment:
Maybe we should also link to the relevant PR https://github.com/SAP/ui5-builder/pull/695 ?